### PR TITLE
Auto update all reviews in the background after parser is updated

### PR DESCRIPTION
**Pull request by praveenkuttappan**
Update automatic reviews also in the background. This is one time update after system restart(which is usually after new deployment)

Automatic reviews are currently attempted to update during auto review creation and this leads to longer delay and request time out. This PR has a change to update automatic reviews in background to latest parser version in the same way manual reviews are handled and avoids attempting to update again during auto review creation.

### CODE REVIEW

**Comment by praveenkuttappan** [comment]
@pakrym : I have removed updating review step as part of this automatic review creation wince this takes longer for few reviews and causes request delay of more than 100 sec. Based on previous background process change, background process doesn't take too longer and this seems like better place to update all reviews(if required for that review)

File: src/dotnet/APIView/APIViewWeb/Repositories/CosmosReviewRepository.cs [hunk 1]
```diff
@@ -62,15 +62,21 @@ public async Task<ReviewModel> GetMasterReviewForPackageAsync(string language, s
             return reviews.FirstOrDefault();
         }
 
-        public async Task<IEnumerable<ReviewModel>> GetReviewsAsync(string language, string packageName, bool isAutomatic)
+        public async Task<IEnumerable<ReviewModel>> GetReviewsAsync(string language, string packageName = "", bool? isAutomatic = null)
         {
             var queryStringAutomaticFilter = "SELECT * FROM Reviews r WHERE " +
```
Comment (pakrym) [inline src/dotnet/APIView/APIViewWeb/Repositories/CosmosReviewRepository.cs:67]: What's the difference between these two queries. It's would be very hard to maintain them

Comment (praveenkuttappan) [reply src/dotnet/APIView/APIViewWeb/Repositories/CosmosReviewRepository.cs:67]: Fixed. I have reverted to single query that works for all scenarios.

**Review by pakrym — APPROVED** [review]

**Comment by pakrym** [comment]
Nice!

**Comment by pakrym** [comment]
We should remove the Update button from markup and all the supporting code.

**Pull request merged**